MARU/MARU/Screens/Library/BookFavoritesViewController.swift

```diff
@@ -116,26 +101,53 @@ extension BookFavoritesViewController {
 }
 
 extension BookFavoritesViewController: UICollectionViewDataSource {
+  func numberOfSections(in collectionView: UICollectionView) -> Int {
+    if data?.bookcase.count ?? 0 % 3 == 0 {
+      return data?.bookcase.count ?? 0 / 3
+    } else if data?.bookcase.count ?? 0 > 3 {
+      return data?.bookcase.count ?? 0 / 3 + 1
+    }
+    return 1
+  }
+
   func collectionView(
     _ collectionView: UICollectionView,
     cellForItemAt indexPath: IndexPath
   ) -> UICollectionViewCell {
     let cell: BookFavoritesShelfCell = collectionView.dequeueReusableCell(forIndexPath: indexPath)
-    cell.rx.binder.onNext((data!))
+    cell.collectionView.backgroundView = bookShelfImageView
+    cell.rx.binder.onNext(data?.bookcase ?? [])
+    cell.rx.didTapContentView
+      .subscribe( onNext: { [weak self] _ in
+        guard let self = self else { return }
+        let viewController = CreateQuizViewController(
+          bookModel: BookModel.init(
+            isbn: self.data?.bookcase[indexPath.item].isbn ?? 0,
+            title: self.data?.bookcase[indexPath.item].title ?? "",
+            author: self.data?.bookcase[indexPath.item].author ?? "",
+            imageURL: self.data?.bookcase[indexPath.item].imageURL ?? "",
+            category: self.data?.bookcase[indexPath.item].category ?? "",
+            hasMyBookcase: true)
+        )
+        if self.data?.bookcase[indexPath.item].canMakeGroup == true {
+          self.navigationController?.pushViewController(viewController, animated: true)
+        } else {
+          // TODO: - 모임열수없음 팝업 추가
+          let targetViewController = JoinLimitViewController()
+          targetViewController.modalPresentationStyle = .overCurrentContext
+          targetViewController.modalTransitionStyle = .crossDissolve
+          self.present(targetViewController, animated: true)
+        }
```

Comment:
```suggestion
      .subscribe( onNext: { [weak self] _ in
        guard 
          let self = self,
          let bookcase = self.data?.bookcase[indexPath.item] 
        else { return }
        let viewController = CreateQuizViewController(
          bookModel: BookModel.init(
            isbn: bookcase.isbn,
            title: bookcase.isbn.title,
            author: bookcase.isbn.author,
            imageURL: bookcase.isbn.imageURL,
            category: bookcase.isbn.category,
            hasMyBookcase: true)
        )
        if bookcase..canMakeGroup == true {
          self.navigationController?.pushViewController(viewController, animated: true)
          return
        }
          // TODO: - 모임열수없음 팝업 추가
          let targetViewController = JoinLimitViewController()
          targetViewController.modalPresentationStyle = .overCurrentContext
          targetViewController.modalTransitionStyle = .crossDissolve
          self.present(targetViewController, animated: true)
```

여기도 이렇게 하면 좀 더 깔끔하게 짤수있을거같아요~

bookcase에 옵셔널인 항목을 모르기 때문에 확인 Plz